src/components/alert/alert-component.ts

```diff
@@ -128,6 +140,29 @@ export class AlertCmp {
     }
   }
 
+  ionViewPreLoad() {
+    if (this.d.subview && this.d.subview.component) {
+
+      const componentFactory = this._cfr.resolveComponentFactory(this.d.subview.component);
+
+      const componentProviders = ReflectiveInjector.resolve([
+        { provide: NavParams, useValue: new NavParams(this.d.subview.args) }
+      ]);
+      const childInjector = ReflectiveInjector.fromResolvedProviders(componentProviders, this._viewport.parentInjector);
+
+      // ******** DOM WRITE ****************
+      const componentRef = this._viewport.createComponent(componentFactory, this._viewport.length, childInjector, []);
+      this.instance = componentRef.instance;
+      this._viewCtrl._setInstance(componentRef.instance);
```

Comment:
Same here, can we add a `const` variable that references `this._viewCtrl`, for example `const viewController = this._viewCtrl`